LICENSE

```diff
@@ -0,0 +1,1170 @@
+The Open Computing Facility's directors and members believe strongly in
+the importance of free and open-source software, and we hope for
+software and other works developed by our organization to be used to
+advance the free software movement.
+
+We have spent a considerable amount of time discussing which licenses
+best reflect our organization's mission, with a roughly even split among
+members who believe that a copyleft license like the GPL best promotes
+the movement's interests, and those who believe that a permissive
+license like the Apache license promotes them better.
+
+The Board of Directors has agreed to dual-license our code under both
+the GPLv3 and the Apache 2.0 licenses. While we recognize that
+dual-licensing our code negates much of the power of copyleft licenses
+like the GPL, we chose to dual-license in an attempt to respect the
+legitimate divide in our organization regarding software licensing.
+
+You may choose to use the code assets in this repository under either
+the GPLv3 or Apache 2.0 license (both printed in full below). The
+non-code assets (text, images, etc.) are licensed under the Creative
+Commons Attribution 4.0 International ("CC BY 4.0") license. A
+human-readable summary of the CC BY 4.0 is available at:
+https://creativecommons.org/licenses/by/4.0/
+
+All copyrights are owned by the original contributors. Contributors are
+asked to sign the Open Computing Facility's CLA, which allows the Board
+to license their contributions.
+
+Note that while we have attempted to identify which files in this
+repository do not contain exclusively OCF contributions (and thus are
+not necessarily licensed under the terms above), it is your
+responsibility to confirm this.
```

Comment:
This is the case with all license grants. Most projects are including code which is not properly licensed/attributed, unfortunately. We can't grant rights to other peoples' work; thus we've done our best to identify it, but if we've messed up, we can't be held responsible for you using it.

The fix is to be extra careful with including others' code and make sure attribution is properly done (as we should/must be doing already). I think our effort is better spent here.